modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -27,13 +25,23 @@ This section also includes four tables that set out the:
 NOTE: In particular, this means {productname} 6.0 does not support Microsoft Internet Explorer 11.
 +
 . Premium Plugins released with {productname} 6.0 are not backwards compatible with earlier versions of {productname}.
-. {productname} 6.0 is now available in three further languages:
+. Core {productname} 6.0 is now licensed under the MIT License.
```

Comment:
This is incorrect, it's only the open source version that's licensed under MIT. The editor core for enterprise downloads is under the our commercial license so this needs fixing.

- [ ] Fix incorrect license information

Reply:
Edited to explicitly note it’s the open source version that is now MIT Licensed.